base/nullable.jl

```diff
@@ -63,16 +63,79 @@ get(x::Nullable) = x.isnull ? throw(NullException()) : x.value
 
 isnull(x::Nullable) = x.isnull
 
-function isequal(x::Nullable, y::Nullable)
-    if x.isnull && y.isnull
-        return true
-    elseif x.isnull || y.isnull
-        return false
+
+## Operators
+
+"""
+    null_safe_op(f::Any, ::Type)::Bool
```

Comment:
Yeah, there may be better names. `uninit` sounds too much like an action, though. Maybe `undef_safe_op`?